design-document.md

```diff
@@ -1,8 +1,89 @@
 # Prio v3 Design Document
 
-## Sample user stories
+## Architecture overview
 
-## Threat model
+Prio is a system and protocol for privately computing aggregation functions over private 
+input. An aggregation function F is one that computes an output y = F(x1,x2,...) for inputs
+xi. In general, Prio supports any aggregation function whose inputs can be encoded in a 
+particular way. However, not all aggregation functions admit an efficient encoding, rendering
+them impractical to implement. Thus, Prio supports a limited set of aggregation functions, 
+some of which we highlight below:
+
+- Simple statistics, including sum, mean, min, max, variance, and standard deviation;
```

Comment:
[[OPEN ISSUE: It's possible to estimate quantiles such as the median. How practical is this?]]